### ISSUE TITLE
Need more rust

### ISSUE THREAD

**Issue by Aeolun** [illacceptanything/illacceptanything#1030]
I don’t see Rust represented anywhere in this repositories languages. There should be at least one component that uses it, but much better would be if we rewrite all the already functional parts in Rust.

It’s faster.

**Comment by AcaiBerii** [comment]
no nemerle, c#, vb.net either!!!!!! 😡😡😠👿🤬🤬🤬🤬

**Comment by chuigda** [comment]
_Technically, Rust does not function good as some functional language. Lacking TCO and unlimited extent making it awkward to use certain functional tricks._

**Comment by Ruzie** [comment]
No, Rust ain't fast, at least not really from C/C++. It's like juice with LLVM backend, and where Rust compiler takes decades to compile a single program. I hate that how funking slow Rust compiler is. I don't need memory safety, and that should be in my control what I can do with memory, I don't need their "safe" code, as `unsafe` exist.

There's only one benefit is Rust compiler and LLVM does a great job, though still, they all do static linking which makes output binary size larger.
